### ISSUE TITLE
remove 1-base input

### ISSUE THREAD

**Issue by hitz** [hitz/RegulomeDB#22]
according to Alan, no one (i.e, UCSC) uses this anymore - except for GFF.  He has already removed this from the front page, so this is just a tracking stub.  Will be in the beta release.

**Comment by euriehong** [comment]
Again, this the difference between computer scientists vs. biologists.  No biologist will know to convert to 0-base or even know what that means.  By removing 1-base, you've effectively made this website IRRELEVANT to biologists.  

This is not the right decision for this website and should not have been made unilaterally.

**Comment by aboyle** [comment]
This was in response to a reviewer complaint that UCSC standard format was considered 1-based instead of 0-based (it should be considered 0-based). It is not true that the 0-based vs 1-based convention creates a website that is irrelevant to biologists. UCSC follows this convention and seems to have stood the test of time. Based on the comments, I thought it better to hold to a single standard for poorly formatted input (just a copy/paste from a browser for example as opposed to BED, VCF, GFF) to best avoid confusion. 

However, if we are insistent that 1-based should still be included, we should move the chr*:start-stop from the 1-based category to the 0-base category. We should also indicate on the results page how the input was interpreted (0-based vs 1-based).

**Comment by euriehong** [comment]
Agreed that we need to streamline the input page and add 0/1-base on the results page.  I'll let it go for the paper but I have some ideas for how to make it more clear.

Trust me, talking to biologists, they have no clue about 0-based vs 1-based.  Since most everything at NCBI is 1-based, they just cut and past coordinates into boxes.  And don't get me started on what most biologists think about the UCSC browser.

**Comment by hitz** [comment]
Eurie - 
can you send an example link from NCBI that has 1-based coordinates that you think people might use/copy-paste?  At least that way we can make sure the format is compat.

**Comment by hitz** [comment]
1-base input option has been removed BUT the docs have not been updated to reflect this.
